package.json

```diff
@@ -45,6 +45,7 @@
     "monocle-ts": "^2.3.13",
     "mustache": "^4.2.0",
     "newtype-ts": "^0.3.5",
+    "ngrok": "^4.3.3",
```

Comment:
You mentioned in Discord discussion that you didn't want ngrok to be present in our production builds - how strongly do you feel about that?

It is, as it turns out, is tricky to arrange. The library is the easiest way to talk to a running ngrok daemon, but it also includes the ngrok daemon (or really, fetches it as a post-install). Making it a dev dependency doesn't work, as Meteor's build system won't let you import a dev dependency in application code:

```
W20221214-18:33:52.911(-8)? (STDERR) (node:41101) UnhandledPromiseRejectionWarning: Error: Cannot find module "/node_modules/ngrok/index.js". Try installing the npm package or make sure it is not a devDependency.
```